### ISSUE TITLE
Telegram is always incomplete

### ISSUE THREAD

**Issue by jellevervloessem** [fliphess/esp8266_p1meter#12]
Hi,

I have your esp8266_p1meter (the version from your development branch) running directly powered by the meter itself but I still have some issues:
- I consistently miss the first lines of the telegram so line 1.0-1.8.1 & 1.0-1.8.2 are not being parsed.
- CRC is never calculated correctly, also because I miss the first lines -> I overruled the CRC validation for testing
- sometimes I read 0 values on the consumption counters (probably also by the missing lines AND no CRC validation)

So basically, I miss the first lines. 
- Could it bet that my pullup resistor has something to do with that? (not that familiar with these things)
- Maybe the interval? it's set to run every 15seconds. I have bad results with 30s and 60s not sending anything at all....

**Comment by fliphess** [comment]
Hey @jellevervloessem ! 

Can you give it a try with the version of @daniel-jong @ https://github.com/daniel-jong/esp8266_p1meter ? 

The code in the development branch is a work-in-progress version based on his changes and focussed on newer meters, but as my meter does not support DSMR version 4.x it's hard for me to work on it and it came a bit to a halt.

(I think I should archive this repository and point people with newer meters to the repo mentioned or remove the development branch and point out in the readme my sketch in master only supports meters with DSMR version 2.0 and 2.2)

I can't be of much help as I'm not much into the inner workings anymore, I've written this sketch years ago when I had the time to spend half a week on a chair next to my meter, but it faded a lot and I'm not using it myself at the moment... 

You could try to connect a serial to usb device to your meter and a laptop to see what the telegram is send and how often and adjust the code accordingly to your liking or try with a faster esp8266 if that's the showstopper (Try a newer D1 instead of a nodemcuv2)
